nylas/client/client.py

```diff
@@ -278,6 +316,67 @@ def availability(
             "start_time": start_time,
             "end_time": end_time,
             "free_busy": free_busy or [],
+            "open_hours": open_hours or [],
```

Comment:
`buffer` and `round_robin` are two more optional inputs that a call can have. 

`buffer` (int) - same as what you have in consecutive availability, non-negative integer representing the number of minutes of buffer time you want around existing meetings.

`round_robin` (string) - accepts "max-fairness" or "max-availability" (or None) as inputs. "max-availability" will find the union of availability among the given emails/free-busy. "max-fairness" will do the same but it will exclude people who were recently booked as a robin through the events endpoint if applicable.

Reply:
Both are now available for `availability`.